app/helpers/distance_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Helpers
+  module DistanceHelper
+    def calculate_euclidean_distance(point_a_x, point_a_y, point_b_x, point_b_y)
```

Comment:
Why define this method as a helper? 

Theoretically modules are for code reuse, but i don't see any other places where this module is included other than CoffeeShopService.

Reply:
The method is defined as a helper to encapsulate the logic for calculating the Euclidean distance in a reusable and modular way, and it keeps the CoffeeShopService class clean and focused on its main task of finding the closest coffee shop, while the helper module handles the specific calculation details.